Deliverable 1/API Examples/reportInteractions.json

```diff
@@ -0,0 +1,15 @@
+{
+  "UI_UX_Request": {
+    "query": "How do I parse a JSON file",
+    "links_ignored": [
+      "link1.com",
+      "link2.com", 
+      "link3.com"
+    ],
+    "link_clicked": "magical.awesome.link.com"
+  },
+
+  "Eval_Response": {
+    "value": "Some sort of acknowledge"
```

Comment:
For all of these, leaving the response as this, not really sure what to do here

Reply:
What is the response supposed to be? Is it our module giving notes?

Reply:
It is telling the sender that we received the message, if there is a problem we can also return an error